plugins/copr.py

```diff
@@ -36,13 +36,13 @@
 # If that fails, attempt to import the deprecated implementation
 # from the platform module.
 try:
-    from distro import linux_distribution as __linux_distro
+    from distro import linux_distribution as linux_distro
```

Comment:
How about removing this `as linux_distro` import and rename the function on the line 45 to `def linux_distribution()`?

Reply:
@clime @xsuchy I am neutral here. What do you think?

Reply:
I made the changes as requested.